tests/utils/trainer_fit.py

```diff
@@ -9,21 +9,25 @@
 from composer.models.base import BaseMosaicModel
 from composer.models.classify_mnist.mnist_hparams import MnistClassifierHparams
 from composer.optim.optimizer_hparams import SGDHparams
+from composer.trainer.ddp import DDP
 from composer.trainer.devices.device_gpu import DeviceGPU
 from composer.trainer.trainer import Trainer
 from composer.trainer.trainer_hparams import TrainerHparams
 from composer.utils import ensure_tuple
 
 
-def get_total_loss(model: BaseMosaicModel, dataloader: DataLoader):
+def get_total_loss(model: BaseMosaicModel, dataloader: DataLoader, ddp: DDP):
     with torch.no_grad():
         total_loss = 0
         for batch in dataloader:
             outputs = model(batch)
             loss = model.loss(outputs, batch=batch)
             for l in ensure_tuple(loss):
                 total_loss += l.item()
-        return total_loss
+
+        total_loss_tensor = torch.Tensor([total_loss])
```

Comment:
Nice good catch

Reply:
Yeah, this was causing a fair amount of flakiness in this test. This change is definitely "correct", but I'm confused why this wasn't causing more flakiness in the previous branch.